hazelcast-client/src/main/java/com/hazelcast/client/spi/impl/ClientExecutionServiceImpl.java

```diff
@@ -142,11 +168,11 @@ public void shutdown() {
         shutdownExecutor("internal", internalExecutor);
     }
 
-    private void executeInternalSafely(Runnable command) {
+    private void executeInternalSafely(Runnable command, ExecutionCallback executionCallback) {
         try {
             executeInternal(command);
```

Comment:
Yes, usage seems not right by itself but my fix was for schedule methods. This method used by them. And if we get an exception when scheduled task trying to call executeInternalSafely, we may miss an exception.